packages/manager/src/lib/requireResolve.ts

```diff
@@ -0,0 +1,30 @@
+import _module, { createRequire } from "node:module";
+
+/**
+ * Resolves a module path with `createRequire().resolve()` with Yarn PnP
+ * support.
+ *
+ * @param id - Module to resolve.
+ * @param from - Location to resolve the module from.
+ *
+ * @returns - Resolved module path.
+ */
+export default (id: string, from: string): string => {
+	let _id = id;
```

Comment:
💡 #idea: `_` prefix usually denotes private data. In this case, `_id` is more like a temporary mutable value. How about naming it `resolvedID`?